.storybook/DocsDecorator/HealthFactors/SingleDoc/index.styles.ts

```diff
@@ -12,13 +12,12 @@ const styles = {
     boxSizing: 'border-box',
   },
 
-  linkIconContainer: (theme: Theme) =>
-    css({
-      width: `${pixelsToRem(14)}rem`,
-      height: `${pixelsToRem(14)}rem`,
-      marginLeft: '0.5em',
-      verticalAlign: `-${pixelsToRem(2)}rem`,
-    }),
+  linkIconContainer: {
+    width: `${pixelsToRem(14)}rem`,
+    height: `${pixelsToRem(14)}rem`,
+    marginLeft: '0.5em',
```

Comment:
Should this be 'em'?

Reply:
Webcore uses em, I am assuming it was a UX requirement on their side to have the spacing depending on font size.